eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -168,7 +230,7 @@ function Add-IssueComment {
 }
 
 # Will add labels to existing labels on the issue
-function Add-IssueLabels {
+function Add-GithubIssueLabels {
```

Comment:
Should this be called `Set-GitHubIssueLabels` so it is clear that this sets all the labels and doesn't just add additional labels?

Reply:
Actually it adds additional labels